File: Algorithms/p168_Excel_Sheet_Column_Title.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string res;
        while(1){
            n--;
            res += 'A'+n%26;
            n = n/26;
            if(!n){
                break;
            }
        }
        reverse(res.begin(),res.end());
        return res;
    }
};

File: Algorithms/p168_Excel_Sheet_Column_Title/p168_Excel_Sheet_Column_Title.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        l = []
        ord_A = ord('A')
        while True:
            n -= 1
            l.append(chr(ord_A+n%26))
            n //= 26
            if not n:
                break
        l.reverse()
        return ''.join(l)
